massgov/openmass
# DP-26237 collapsable content labels

**Description:**
Explain the technical implementation of the work done.


**Jira:** (Skip unless you are MA staff)
DP-26237


**To Test:**
- [ ] Test sample page:  /pandemic-unemployment-assistance-pua
See https://github.com/massgov/mayflower/pull/1703 for testing.


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
